src/components/v5/common/ActionSidebar/ActionSidebar.tsx

```diff
@@ -109,12 +110,32 @@ const ActionSidebar: FC<PropsWithChildren<ActionSidebarProps>> = ({
   const actionNotFound = transactionId && !action;
 
   const getSidebarContent = () => {
-    if (isLoading) {
+    if (isLoading && !action) {
       return <ActionSidebarLoadingSkeleton />;
     }
 
     if (action) {
-      return <CompletedAction action={action} />;
+      return (
```

Comment:
could we just return the skeleton `if (isLoading || motionsLoading)` otherwise return the `CompletedAction`?
It's a bit funky to render both at the same time just hide them with CSS